server/snapshot/Dockerfile

```diff
@@ -47,6 +48,18 @@ RUN SNAPSHOT_VERSION=$(curl --silent -fSL $MAVEN_OSS_SNAPSHOT/io/debezium/debezi
 #
 RUN chmod -R g+w,o+w $SERVER_HOME
 
+# Stage 2: Final image
+FROM registry.access.redhat.com/ubi8/openjdk-11
+
+ENV SERVER_HOME=/debezium
```

Comment:
We'd need to test but if we need to redefine `SERVER_HOME` here for this multi-build process then I would probably suggest we should also consider redefining the other two environment variables for `DEBEZIUM_VERSION` and `MAVEN_OSS_SNAPSHOT` just on the slight chance that someone maybe is using the images and expect these variables to be defined in the container to avoid any potential image regressions.  wdyt?